ocaml/test/test_pool_update.ml

```diff
@@ -30,9 +31,21 @@ let test_pool_update_refcount () =
 	Xapi_pool_update.with_dec_refcount ~__context ~uuid ~vdi (fun ~__context ~uuid ~vdi -> assert_equal 0 1);
 	Xapi_pool_update.with_dec_refcount ~__context ~uuid ~vdi (fun ~__context ~uuid ~vdi -> ())
 
+let test_assert_space_available () =
+	let stat = statvfs Xapi_globs.host_update_dir in
+	let free_bytes = Int64.mul stat.f_frsize stat.f_bfree in
+	Xapi_pool_update.assert_space_available (Int64.div free_bytes 3L);
+	try 
+		Xapi_pool_update.assert_space_available (Int64.div free_bytes 2L);
+		assert_equal 0 1
+	with 
+	| Api_errors.Server_error(code, _) when code=Api_errors.out_of_space -> ()
```

Comment:
I'm not an `OUnit` expert but could this be better implemented with `assert_raise`? `OUnit.assert_raise` is for testing that an exception is raised as expected. Also, could you have matched `Api_errors.Server_error(Api_errors.out_of_space)` instead of using `when`?